src/topology/algebra/ring.lean

```diff
@@ -10,6 +10,25 @@ import ring_theory.ideal.basic
 import ring_theory.subring
 import algebra.ring.prod
 
+/-!
+
+# Topological (semi)rings
+
+A topological (semi)ring is a (semi)ring equipped with a topology such that all operations are
+continuous. Besides this definition, this file proves that the topological closure of a subring
+(resp. an ideal) is a subring (resp. an ideal) and defines products and quotients
+of topological (semi)rings.
+
+## Main Results:
```

Comment:
Minor point: remove the semicolon to match the style of other docs.

Reply:
what do you mean - do other docs not use semicolons?

Reply:
```suggestion
## Main Results
```

Oh I see